packages/tasks/src/pipelines.ts

```diff
@@ -550,6 +550,11 @@ export const PIPELINE_DATA = {
 		color: "blue",
 		hideInModels: true,
 	},
+	"text-ranking": {
+		name: "Sentence Ranking",
```

Comment:
Shouldn't it be "Text Ranking" instead? (based on https://github.com/huggingface-internal/moon-landing/issues/12682#issuecomment-2682758590)

```suggestion
		name: "Text Ranking",
```

Reply:
Yes! Thanks for spotting this, I forgot to replace these when I locally renamed the tag from `sentence-ranking` to `text-ranking`. Resolved in 4865124b131a62813988f63cc28e9f40fc3994d0.